src/polyfills.ts

```diff
@@ -19,7 +19,7 @@
  */
 
 /** IE10 and IE11 requires the following for NgClass support on SVG elements */
-import "classlist.js"; // Run `npm install --save classlist.js`.
+// import "classlist.js"; // Run `npm install --save classlist.js`.
```

Comment:
please remove commented out code

Reply:
reverted back to your initial commit. This is how the default pollyfills.js looks

Reply:
oh i see. better to remove the commented out code tho even if this is how the default pollyfills.ts is

Reply:
Could do, the pollyfills.ts file is full of commented out code and instructions though? I think we should leave as default for future dev requirements?